editoast/justfile

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env just --justfile
+
+# Display available recipes
+help:
+    @just --list --unsorted
+
+# Run the application; does not start other services
+run:
+    cargo run
+
+# Install required dependencies
+install:
+    cargo build
+
+# Run the tests
+test:
+    cargo test --workspace -- --test-threads=4
+
+# Format the source code
+format:
+    cargo fmt
```

Mainly adding taplo (ditto for all cargos):
```suggestion
    cargo fmt --all
    taplo fmt
```